test/functions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require('../functions.php');
+use PHPUnit\Framework\TestCase;
+class collection_appTest extends TestCase
+{
+    public function testSuccessDBCheckIntInput () {
+        $expected = 1;
+        $this->expectOutputString($expected);
+        DBCheck(1);
```

Comment:
because this function echos, you have to use the output string thing which isnt very good, and results in "risky" tests.